tests/functional_tests/0003_create_test.py

```diff
@@ -161,27 +156,46 @@ def test_09_test_dhcp_connectivity_in_jail(release, jail, invoke_cli):
     ip = jail.ip
     assert ip is not None and '0.0.0.0' not in ip
 
+    # Let's test if we can ping 8.8.8.8 from inside the jail
+
+    stdout, stderr = jail.run_command(['ping', '-c', '5', '8.8.8.8'])
```

Comment:
Perhaps we should allow this to be configurable, there may be users who do not want Google to be aware of each test :D